components/workoutDay/WorkoutDay.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components"
+
+const StyledSection = styled.section`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    box-shadow: 1px 1px 2px 2px rgba(0, 0, 0, 0.1);
+    max-width: fit-content;
```

Comment:
```suggestion
    width: fit-content;
```

Reply:
In this instance, you don't need the "max-", because either property leads to the same result.